app/controllers/gobierto_data/api/v1/datasets_controller.rb

```diff
@@ -11,6 +11,48 @@ class DatasetsController < BaseController
         skip_before_action :authenticate_in_site, only: [:new, :create, :update, :destroy]
         skip_before_action :set_admin_with_token, except: [:new, :create, :update, :destroy]
 
+        def catalog
+          # TODO TRANSLATIONS...
+          @catalog_identifier_uri = request.base_url + request.path
+          @catalog_title = "dcat catalog for #{current_site}"
+          @catalog_description = "this is catalog published by #{@catalog_identifier_uri} which contains datasets created by"
+          @catalog_issued = current_site.created_at
+          @catalog_modified = GobiertoData::Dataset.maximum(:created_at) || current_site.created_at
```

Comment:
use :updated_at

Reply:
I was thinking  about the when it is considered that catalog was be updated 
 catalog update != dataset modifications

- initialization site:
it implicate new catalog (despite that catalog was empty of datasets)

- adding more datasets:
for example: when new dataset was created the catalog should be consider catalog updated because

- removing dataset:
this is more difficult without have a model persisted dataset because deletion of dataset can't be tracked

Reply:
for catalog updated use site.updated_at, because when datasets change there's a touch that updates the site updated_at to remove cache